ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -806,6 +811,47 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the cluster peer's IPFS repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) (*api.RepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	repoGC := api.RepoGC{
+		Keys: make([]api.IPFSRepoGC, 0),
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, ipfs.config.RepoGCTimeout)
+	defer cancel()
+
+	res, err := ipfs.doPostCtx(ctx, ipfs.client, ipfs.apiURL(), "repo/gc?stream-errors=true", "", nil)
+	if err != nil {
+		logger.Error(err)
+		return &repoGC, err
```

Comment:
return nil,err

You can move the `repoGC` declaration down, closer where it is used.